python/az/aro/azext_aro/custom.py

```diff
@@ -290,6 +290,23 @@ def get_route_tables_from_subnets(cli_ctx, subnets):
     return route_tables
 
 
+def get_nat_gateways_from_subnets(cli_ctx, subnets):
```

Comment:
The `get_route_tables_from_subnets` function is doing the exact same thing but checking `subnet.route_table` instead of `subnet.nat_gateway`.  Can we dry this up and have a single function which returns both a set of `route_tables` and `nat_gateways`? 

Could name it `get_network_resources_from_subnets` or something like that.

Reply:
Done